lib/routes/devices-sessions.js

```diff
@@ -339,41 +339,41 @@ module.exports = (log, db, config, customs, push, devices) => {
         db.sessions(uid)
           .then(sessions => {
             reply(sessions.map(session => {
-              session.id = session.tokenId
-              // if session has a device record
-              session.isDevice = !! session.deviceId
+              const deviceId = session.deviceId
+              const isDevice = !! deviceId
 
-              if (! session.deviceName) {
-                session.deviceName = devices.synthesizeName(session)
+              let deviceName = session.deviceName
+              if (! deviceName) {
+                deviceName = devices.synthesizeName(session)
               }
 
-              session.userAgent = (session.uaBrowser + ' ' + session.uaBrowserVersion).trim()
-
-              if (! session.deviceType) {
-                session.deviceType = session.uaDeviceType || 'desktop'
+              let userAgent
+              if (! session.uaBrowser) {
+                userAgent = ''
+              } else if (! session.uaBrowserVersion) {
+                userAgent = session.uaBrowser
```

Comment:
> I'm not sure what the reasons are on line 356 for trimming the result, does uaBrowser need to be trimmed to?

Actually the `trim()` is unnecessary now that we check whether `uaBrowser` and `uaBrowserVersion` before concatenating. Will remove it, thanks.